tests/types/types.go

```diff
@@ -194,6 +195,15 @@ func GetBaseDisk() []Disk {
 					Length:         262144,
 					FilesystemType: "ext4",
 				},
+				{
+					Number:         12,
+					Label:          "READ-ONLY",
```

Comment:
This adds a new partition to every single test, which isn't necessary, and seems like it might affect other tests.  Let's just add the affected partition (or, more likely, reconfigure an existing one) in the new tests.

Reply:
Makes sense ill drop in the next push!
ill have a go at re configuring an existing disk.